packages/acceptance-tests/pkg-tests-specs/sources/commands/workspaces/foreach.test.js

```diff
@@ -469,6 +470,67 @@ describe(`Commands`, () => {
         },
       ),
     );
+
+    test(
```

Comment:
Both this test file and the one for list are missing tests with `--recursive`!

Relatedly, would it make sense to include tests that highlight the difference between specifying `--recursive` and not when used on a workspace that has child workspaces?

Reply:
Added some initial tests for `--recursive` in 711507b73bf4475858bdcd05d4d4aa4bb6bd5335.
Will add some tests to cover the cases you suggested in my next commit.